### PR TITLE
Update smart contract page

### LINKED ISSUE #157: Update smart contract adresses
### Description

Add newly deployed Mainnet contract addresses and review existing ones.

### Location
https://docs.lisk.com/contracts

### Additional Information
https://github.com/LiskHQ/lisk-contracts/blob/development/deployment/addresses/mainnet/addresses.md

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
### What was the problem?

This PR resolves #157 

### How was it tested?

https://liskhq.github.io/lisk-documentation/contracts

### CODE REVIEW

File: docs/building-on-lisk/contracts.mdx [hunk 1]
```diff
@@ -0,0 +1,604 @@
...
+    <td>[0x76f1cD8436373fa9f3c17Da1e39740fE9dB9a04B](https://sepolia.etherscan.io/address/0x76f1cd8436373fa9f3c17da1e39740fe9db9a04b)</td>
+    <td>🟡</td>
+    </tr>
+  </tbody>
+  <tfoot>
+  </tfoot>
+</table>
+
+
+### Vesting Wallets (L1)
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.mdx]: Vesting wallets should not be included here, please remove

File: docs/building-on-lisk/contracts.mdx [hunk 2]
```diff
@@ -0,0 +1,604 @@
...
+    <td>[0x841e828A69B6efC1b02F7C317F59291A39583a64](https://sepolia-blockscout.lisk.com/address/0x841e828A69B6efC1b02F7C317F59291A39583a64)</td>
+    <td>🟣</td>
+    </tr>
+
+  </tbody>
+  <tfoot>
+  </tfoot>
+</table>
+
+### Vesting Wallets (L2)
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.mdx]: Same as above, Vesting wallets should not be listed here

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/contracts.mdx (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/contracts.mdx]: Why? Aren't these smart contracts as well?

Comment (Tschakki) [reply docs/building-on-lisk/contracts.mdx]: Technically yes, but this is not relevant for devs to know the vesting wallets of the team for example

File: docs/building-on-lisk/contracts.mdx [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+    <td>[0x841e828A69B6efC1b02F7C317F59291A39583a64](https://sepolia-blockscout.lisk.com/address/0x841e828A69B6efC1b02F7C317F59291A39583a64)</td>
+    <td>🟣</td>
+    </tr>
+
+  </tbody>
+  <tfoot>
+  </tfoot>
+</table>
+
+### ERC-20 tokens deployed on Lisk L2
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.mdx]:
```suggestion
### ERC-20 tokens deployed on Lisk
```

File: docs/building-on-lisk/contracts.mdx [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+    <tr>
+    <td>[0x4200000000000000000000000000000000000018](https://sepolia-blockscout.lisk.com/address/0x4200000000000000000000000000000000000018)</td>
+    <td>🟣</td>
+    </tr>
+  </tbody>
+  <tfoot>
+  </tfoot>
+</table>
+
+### Lisk features-specific contracts (L2)
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.mdx]:
```suggestion
### Lisk features-specific contracts
```

File: docs/building-on-lisk/contracts.mdx [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+    </tr>
+  </tbody>
+  <tfoot>
+  </tfoot>
+</table>
+
+## Lisk network (L2)
+
+
+### Predeploys (L2)
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.mdx]:
```suggestion
### Predeploys on Lisk
```

File: docs/building-on-lisk/contracts.mdx (hunk 4 above)
Comment (Tschakki) [reply docs/building-on-lisk/contracts.mdx]: would remove the L2 because it is already mentioned in parent section

**Review by Tschakki — APPROVED** [review]
LGTM, just left some small suggestions 👍

File: docs/building-on-lisk/contracts.mdx [hunk 6]
```diff
@@ -0,0 +1,479 @@
...
+      <td>🟢</td>
+    </tr>
+    <tr>
+    <td>[0x27Bb4A7cd8FB20cb816BF4Aac668BF841bb3D5d3](https://sepolia.etherscan.io/address/0x27Bb4A7cd8FB20cb816BF4Aac668BF841bb3D5d3)</td>
+    <td>🟡</td>
+    </tr>
+    
+    <tr>
+      <th scope="row" rowSpan="2">L1ERC721Bridge</th>
+      <td scope="row" rowSpan="2">Handles the sending and receiving of messages between L1 and L2.</td>
```
Comment (nagdahimanshu) [inline docs/building-on-lisk/contracts.mdx]: Isn't it handles the sending and receiving of ERC721 tokens from L1 to L2?

File: docs/building-on-lisk/contracts.mdx [hunk 7]
```diff
@@ -0,0 +1,479 @@
...
+      <td>🔵</td>
+    </tr>
+    <tr>
+    <td>[0xDdE1998B8842b1C925Eef7eDFDAA2Df24A75048a](https://sepolia-blockscout.lisk.com/address/0xDdE1998B8842b1C925Eef7eDFDAA2Df24A75048a)</td>
+    <td>🟣</td>
+    </tr>
+
+    <tr>
+      <th scope="row" rowSpan="2">L2Claim <br/>(Proxy)</th>
+       <td scope="row" rowSpan="2"> Proxy contract for claiming LSK tokens from Lisk Chain on L2.</td>
```
Comment (nagdahimanshu) [inline docs/building-on-lisk/contracts.mdx]: Isn't it is better to mention Lisk chain (L1 or v4) here?

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**